src/serialization/font.proto

```diff
@@ -0,0 +1,40 @@
+syntax="proto3";
+// ensure you're using the protobuf 3 syntax
+// get protobut 3 from here on ubuntu
+// https://gist.github.com/sofyanhadia/37787e5ed098c97919b8c593f0ec44d8
+
+// stack all your protobuf structs
+// start from the most base object: the x,y coord 
+message Font {	
+    int32 w = 1;
+	int32 x = 2;
+    int32 y = 3;
+	int32 z = 4;
+    bytes b = 5;
+
+}
+
+// three Points define a BezCurve
+/*
+message BezCurve {
+	Point stt = 1;
+	Point off = 2;
+	Point end = 3;
+}
+
+// repeated curves make a counter space
+message CounterSpace {
```

Comment:
I won't insist on this since all the data has been made, but I'd prefer we call it "closed curve" instead of "counter space". Technically a counter refers to the negatived space _enclosed_ by a closed curve.

Reply:
Resolving this now.